test/spec/algoliasearch.helper/searchForFacetValues.js

```diff
@@ -2,13 +2,22 @@
 
 var algoliasearchHelper = require('../../../index');
 
+function createFakeSearchForFacetValuesResponse() {
```

Comment:
```suggestion
function makeFakeSearchForFacetValuesResponse() {
```

The codebase mainly uses `make` rather than `create`.